### DISCUSSION TITLE
Pool Resource

### DISCUSSION THREAD

**Discussion by luis-n4489** [concourse/concourse#8088] [General]
I have gotten this error before, and fixed it BUT of course did not put a note on what I did to do so. Anyone have any input? I remember it being an issue in the way i created the empty branch and files. The structure is correct in folder / files but again i get. Any help is gladly appreciated 

```
Cloning into '/tmp/build/get'...
2e1dd82 test
fatal: ambiguous argument 'HEAD~1': unknown revision or path not in the working tree.
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'
basename: missing operand
Try 'basename --help' for more information.
```

**Comment by andy-paine** [comment]
Normally `HEAD~1` is "one commit before the current one" - if you only have a single commit in your tree that might fail. You might have to just make an empty commit or something to make the history at least 2 commits long 😬  It looks like this _used_ to be an issue and has since been fixed https://github.com/concourse/pool-resource/issues/54 - what version of Concourse/the pool resource are you on?

**Reply by luis-n4489** [reply]
Yeah I saw that and thought that the latest version concourse would have that update. let me try that out @andy-paine , thank you!
